src/features/home-page/weather/show-weather.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+const KEY = 'f5672b9d8248bed28c43b00460aa62eb';
+const URL = 'https://api.openweathermap.org/data/2.5';
+const CITY = 'LVIV';
```

Comment:
@IraLukianovets move this consts to api folder (create api folder for openweathermap).
Also change logic to get weather depending on current location, not hardcoded.